hw/ip/adc_ctrl/README.md

```diff
@@ -1,6 +1,10 @@
 # Analog to Digital Converter Control Interface
 
-{{#block-dashboard adc_ctrl}}
+[`adc_ctrl`](https://reports.opentitan.org/hw/ip/adc_ctrl/dv/latest/report.html):
+![](https://dashboards.lowrisc.org/badges/dv/adc_ctrl/test.svg)
```

Comment:
Is there any alt text we can to these badges for accessibility? e.g. `[adc_ctrl tota tests passing](...)` etc.

Reply:
The SVG itself is accessible.